st2api/st2api/cmd/api.py

```diff
@@ -64,10 +65,25 @@ def _run_server():
     host = cfg.CONF.api.host
     port = cfg.CONF.api.port
 
-    LOG.info('(PID=%s) ST2 API is serving on http://%s:%s.', os.getpid(), host, port)
+    api_host = cfg.CONF.api.host
+    api_port = cfg.CONF.api.port
 
-    wsgi.server(eventlet.listen((host, port)), app.setup_app())
-    return 0
+    webui_host = api_host
+    webui_port = int(api_port) + 1
+
+    LOG.info('(PID=%s) ST2 API is serving on http://%s:%s.', os.getpid(), api_host, api_port)
+    LOG.info('(PID=%s) ST2 WebUI is serving on http://%s:%s.', os.getpid(), webui_host, webui_port)
+
+    def _run_api_server():
+        wsgi.server(eventlet.listen((api_host, api_port)), get_api_app())
+
+    def _run_webui_server():
```

Comment:
To clarify it - if we want to serve it as part of the api server, we need to set debug=True which is less than ideal to put it lightly.